### PR TITLE
Fixing unescaped hash signs in a Markdown tab.

### PR DESCRIPTION
**Pull request by JayBeale**
The text within these tabs are parsed as Markdown, so the # hash signs were breaking the markdown and rendering the page differently than intended.  I've escaped the hash signs to fix this.

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 1]
```diff
@@ -56,13 +56,13 @@ The upgrade workflow at high level is the following:
     {{% tab name="Ubuntu, Debian or HypriotOS" %}}
     apt update
     apt-cache madison kubeadm
-    # find the latest 1.18 version in the list
-    # it should look like 1.18.x-00, where x is the latest patch
+    \# find the latest 1.18 version in the list
+    \# it should look like 1.18.x-00, where x is the latest patch
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:60]: A better fix: wrap these in triple backticks and mark them as shell.

**Comment by kbhawkey** [comment]
@JayBeale As mentioned by @sftim, would you investigate using a code block?
After changing, feel free to remove the hold.

**Comment by JayBeale** [comment]
I apologize for leaving this pull request open - I had missed coming back to it.  I'll close it.

**Pull request closed without merging**